libs/permission-checker/tests/Checker/Notification/CanModifySubscriptionsTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PersmissionChecker\Tests\Checker\Notification;
+
+use Keboola\PermissionChecker\BranchType;
+use Keboola\PermissionChecker\Checker\Notification\CanModifySubscriptions;
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+use Keboola\PermissionChecker\StorageApiToken;
+use PHPUnit\Framework\TestCase;
+
+class CanModifySubscriptionsTest extends TestCase
+{
+    public static function provideValidPermissionsCheckData(): iterable
+    {
+        yield 'regular token on regular project default branch' => [
+            'branchType' => BranchType::DEFAULT,
+            'token' => new StorageApiToken(role: null),
+        ];
+
+        yield 'regular token on regular project dev branch' => [
+            'branchType' => BranchType::DEV,
+            'token' => new StorageApiToken(role: null),
+        ];
+
+        yield 'productionManager on protected default branch' => [
```

Comment:
tady by teda ale měla být feature

Reply:
uz tam je, asi si stihl jeste starsi verzi :) doplnoval jsem infection, ktery to odhalilo

Reply:
aha, to je od něj moc hezký!